lambdas/service/app.py

```diff
@@ -808,13 +842,6 @@ def files_proxy(uuid):
         dss_response.raise_for_status()
 
 
-def file_url(uuid, **params):
```

Comment:
Inconsequential change. Just changes the order of two methods. Remove.

Reply:
It was a rebasing issue. It has been moved back to this location.